docs/storage/disks_and_volumes.md

```diff
@@ -2131,7 +2131,7 @@ The following strategies are supported:
   * `Replacement`: the update volumes will be replaced upon the VM restart.
   * `Migration`: the update of the volumes will trigger a storage migration of
     the old volumes to the new ones. More details about volume migration can be
-    found in the [volume migration documentation](docs/storage/volume_migration.md).
+    found in the [volume migration documentation]((/storage/volume_migration/).
```

Comment:
Apologies, I made a mistake in the suggestion. It actually requires a different style, which is then rendered out by the generator I believe.

```suggestion
    found in the [volume migration documentation](../storage/volume_migration.md).
```

Please see examples here:
https://github.com/kubevirt/user-guide/blob/dde7f54b2b4e24967f21b2f765e6e1483c60e2eb/docs/cluster_admin/confidential_computing.md?plain=1#L15-L16